Formula/xcftools.rb

```diff
@@ -0,0 +1,52 @@
+class Xcftools < Formula
+  desc "Tools for GIMP's native file format XCF"
+  homepage "http://henning.makholm.net/software"
+  url "http://henning.makholm.net/xcftools/xcftools-1.0.7.tar.gz"
+  sha256 "1ebf6d8405348600bc551712d9e4f7c33cc83e416804709f68d0700afde920a6"
+
+  depends_on "libpng"
+
+  def install
+    # Avoid deprecated use of libpng 1.2
+    inreplace "xcf2png.c", /png_(voidp|error_ptr)_NULL/, "NULL"
+    # Avoid error from empty filename in touch
+    inreplace "Makefile.in", "touch $^ ;", ""
```

Comment:
Instead of modifying the code, I made another workaround by running `make` to skip unnecessary steps. These are only required for NLS-enabled environment and this formula does not depend on `gettext`.